### DISCUSSION TITLE
Worried About PR not being merged

### DISCUSSION THREAD

**Discussion by Kpraful** [TheAlgorithms/Python#10184] [General]
Should we be worried that some of our PR that have already pass the test will not be merged and hence will not be counted in Hacktober contribution or can we just assume at some point it is going to merge so we can continue giving the PR. I understand there are overwhelming PR and It is difficult to merge them all but just  concern about contribution.

**Comment by AaryaNale** [comment]
Well, same, I'm worried about mine too. I thought there was something wrong with my PR?
